lib/livingstyleguide/templates/scripts/search.html.erb

```diff
@@ -0,0 +1,67 @@
+<script>
```

Comment:
Why not removing this `<script>` HTML tag and renaming the file to *search.js.erb?*